python/orca/example/learn/tf/inception/inception.py

```diff
@@ -185,6 +185,8 @@ def config_option_parser():
     parser.add_option("--memory", type=str, default="10g",
                       help="The memory you want to use on each node. "
                            "You can change it depending on your own cluster setting.")
+    parser.add_option('--py_files', type=str, default=None,
+                      help='The pyfiles you need.')
```

Comment:
pyfiles to `python files`

Reply:
Updated in this [commit](https://github.com/intel-analytics/BigDL/pull/3756/commits/219748af9d1cb55fa50c754000c7e34986ea7da6).